### ISSUE TITLE
swap highlight query order

### ISSUE THREAD

**Issue by MrBrN197** [helix-editor/helix#10791]
not sure but this zig highlight queries seems to in the wrong order.
I believe the intention was for the second query to be an exception for the first `BUILTINIDENTIFIER` and swapping the order gets it working as such for me.

https://github.com/helix-editor/helix/blob/2301430e37a98192de7864faa2d8c1b84860f400/runtime/queries/zig/highlights.scm#L69-L72

**Comment by the-mikedavis** [comment]
Yep that is reversed. Feel free to send a PR but we will eventually swap the order that we use so that we match the tree-sitter CLI and neovim (https://github.com/helix-editor/helix/pull/9458) and then we will want to switch the ordering back

**Comment by MrBrN197** [comment]
no problem waiting for that issue is better.
